tests/test_helpers.py

```diff
@@ -32,6 +32,27 @@
       "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
       "--title", "On the roundness of the Earth", "--status", "in-prep"],
      "2006as_mars has been added in presentations\n"),
+    (["helper", "l_progress", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+     ),
+    (["helper", "l_progress", "-v", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+      "    log_url: https://docs.google.com/document/d/1YC_wtW5Q\n"
+      "    team:\n"
+      "        lead: ascopatz\n"
+      "        group_members: ascopatz\n"
+      "        collaborators: aeinstein, pdirac\n"
+      "    milestones:\n"
+      "        Project lead presentation\n"
```

Comment:
let's typeset this with the date leading as this is what I want to pay attention to, so maybe
```
milestones:
    2020-05-20: Project lead presentation
        description: Longer description if it is there
        notes: 
            - only put this in if notes are there
        status: proposed

Reply:
Done. 
There's no "notes" for milestones, so I use "objective" instead.